tests/SignersTests/V4Tests.elm

```diff
@@ -0,0 +1,209 @@
+module SignersTests.V4Tests exposing (all)
+
+import AWS.Config exposing (..)
+import AWS.Http exposing (unsignedRequest, UnsignedRequest, RequestParams(..))
+import AWS.Signers.Canonical exposing (canonical, canonicalRaw)
+import AWS.Signers.V4 as V4 exposing (..)
+import Date exposing (fromString, fromTime, Date)
+import Expect
+import Json.Decode as JD
+import Test exposing (describe, test, Test)
+
+
+all : Test
+all =
+    describe "Signer"
+        [ authorizationTests
+        , awsOfficialSuite
+        ]
+
+
+authorizationTests : Test
+authorizationTests =
+    describe "authorization"
+        {- http://docs.aws.amazon.com/general/latest/gr/signature-v4-test-suite.html#signature-v4-test-suite-example -}
+        [ test "returns a signed header equal to the online example" <|
```

Comment:
@ngmiller I left this one in. From the comment it looks like you got it from the same test suite that I'm adding now. If guess eventually we'll replace this with an entry in the `allData` list